tests/api/views/tracking/latest_test.py

```diff
@@ -0,0 +1,40 @@
+from mock import patch
+from datetime import datetime, timedelta
+from tests.data import add_fixtures, users, live_fix
+
+
+def test_get_latest_default_max_age(db_session, client):
+    """The default max_age is 6 hours"""
+    utcnow = datetime(year=2020, month=12, day=20, hour=12)
+
+    # Only the latest_fix should be returned
+    john = users.john()
+    fix = live_fix.create(john, utcnow - timedelta(hours=2), 10, 20)
+    latest_fix = live_fix.create(john, utcnow - timedelta(minutes=5), 11, 21)
+
+    # Fix older than 6h should not be returned
+    jane = users.jane()
+    old_fix = live_fix.create(jane, utcnow - timedelta(hours=7), 12, 22)
+
+    add_fixtures(db_session, john, fix, latest_fix, jane, old_fix)
+
+    with patch("skylines.model.tracking.datetime") as datetime_mock:
+        datetime_mock.utcnow.return_value = utcnow
+
+        res = client.get("/tracking/latest.json")
```

Comment:
looking at this I was wondering if any of our code was actually using this API endpoint at all and I couldn't find anything. might be worth it to just remove it instead of keeping it alive 😅

Reply:
Hey @Turbo87 I am not exactly sure what you mean here ?

If by "my code" you mean flyxc then I am not using it as of now, that's right.
However I think @indyflyersoft is using it.

Did I get your question correctly ?

Reply:
what I meant was that the endpoint is not used in the skylines frontend code itself

Reply:
OOps "our code" I read too fast. Thanks for the clarification.

Might be worth checking the server logs to see how often it is used.

I think it's handy to have it.
For example flyxc could request the pilot's positions (`/live/<user_ids>`) only if they are listed here.
And on the other hand there is no real reason to remove this.

Just my 2 cents. I don't have strong opinions on whether keeping/dropping this one.